applications/luci-app-apinger/Makefile

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (C) 2022 Jaymin Patel <jem.patel@gmail.com>
+#
+# This is free software, licensed under the Apache License, Version 2.0 .
+#
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=LuCI support for the Apinger
+LUCI_DEPENDS:=+apinger
+LUCI_PKGARCH:=all
+PKG_LICENSE:=GPLv2
```

Comment:
Sorry, but can you use a valid `[SPDX](https://spdx.org/licenses/)` here?
maybe `PKG_LICENSE:=GPL-2.0` because `PKD_LICENSE:=GPLv2` is not valid

Reply:
Updated.